util/src/main/java/org/datacommons/util/StatChecker.java

```diff
@@ -272,21 +274,39 @@ protected static void checkPercentFluctuations(
         } else {
           currDelta = (currVal - baseVal) / Math.abs(baseVal);
         }
-        if (Math.abs(maxDelta) < Math.abs(currDelta)) {
-          maxDelta = currDelta;
-          maxDeltaDP = dp;
-          maxDeltaBaseDP = baseDataPoint;
+        if (Math.abs(currDelta) > 500) {
+          // For these percent fluctuation counters, just show the details for the max fluctuation.
+          if (!percent500Counter.hasPercentDifference()
+              || Math.abs(currDelta) > Math.abs(percent500Counter.getPercentDifference())) {
+            percent500Counter.addProblemPoints(baseDataPoint);
+            percent500Counter.addProblemPoints(dp);
+            percent500Counter.setPercentDifference(currDelta);
+          }
+        } else if (Math.abs(currDelta) > 100) {
+          if (!percent100Counter.hasPercentDifference()
+              || Math.abs(currDelta) > Math.abs(percent100Counter.getPercentDifference())) {
+            percent100Counter.addProblemPoints(baseDataPoint);
+            percent100Counter.addProblemPoints(dp);
+            percent100Counter.setPercentDifference(currDelta);
+          }
+        } else if (Math.abs(currDelta) > 50) {
+          if (!percent50Counter.hasPercentDifference()
+              || Math.abs(currDelta) > Math.abs(percent50Counter.getPercentDifference())) {
+            percent50Counter.addProblemPoints(baseDataPoint);
+            percent50Counter.addProblemPoints(dp);
+            percent50Counter.setPercentDifference(currDelta);
+          }
```

Comment:
Just wondering, why not compute the maxDelta like before, and include a single counter for the series? If there is a point with >500%, just an exemplar pair of points should be sufficient (given this will likely be used to spot-check the outliers)

Reply:
Oh, I thought we wanted counters for each of the percent fluctuations >50, >100, >500? Or do we just want the max percent fluctuation?

Reply:
Just max percent fluctuation. One series should map to a single such counter, so by looking at the counters users can tell how many series have 5x, vs. 0.5x fluctuations... (we might fine-tune the 50, 100, 500 buckets in future)

Reply:
ah, ok, makes sense, updated